src/components/About/AboutCountry.js

```diff
@@ -47,10 +47,7 @@ const styles = theme => ({
 
 function AboutCountry({ classes, dominion }) {
   const { selectedCountry = {} } = dominion;
-  const info = config.about[selectedCountry.slug] || {
-    intro: 'Not found',
-    other: 'Not found'
-  };
+  const info = config.about[selectedCountry.slug] || {};
```

Comment:
If we're sure `info` will always be there, lets remove the ` || {};` as well.

Reply:
I meant if we're sure `config.about[selectedCountry.slug]` will always contain `info`, then there is no need to add ` || {}` to it.